rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -403,25 +411,42 @@ def _deserialize_channel_withdraw(
     ) -> ChannelWithdraw:
         """Deserialize a channel withdraw event. Only for Tom pool.
 
-        It may raise ChannelTokenAddressError, KeyError and ValueError.
+        May raise DeserializationError.
         """
         inverse_identity_address_map = {
             address: identity for identity, address in identity_address_map.items()
         }
-        tx_hash, _, log_index = raw_event['id'].split(':')
-        address = to_checksum_address(raw_event['user'])
-        identity_address = inverse_identity_address_map[address]
-        amount = FVal(raw_event['amount']) / ADX_AMOUNT_MANTISSA
-        channel_id = raw_event['channel']['channelId']
-        token_address = to_checksum_address(raw_event['channel']['tokenAddr'])
+        try:
+            tx_hash, _, log_index = raw_event['id'].split(':')
+            address = to_checksum_address(raw_event['user'])
+            identity_address = inverse_identity_address_map[address]
+            amount = FVal(raw_event['amount']) / ADX_AMOUNT_MANTISSA
+            channel_id = raw_event['channel']['channelId']
+            token_address = to_checksum_address(raw_event['channel']['tokenAddr'])
```

Comment:
below at the end of the function you call `raw_event['timestamp']`. Why is this not in here? A key error from that would propagate out of here